### PR TITLE
Task 7

### PR DESCRIPTION
**Pull request by noway36**
Total 6/6
[FE PR](https://github.com/noway36/shop-react-redux-cloudfront/pull/4)
[Products Import Page](https://d1mj375gxygqlt.cloudfront.net/admin/products)
Cognito auth (no evaluation) is still in progress

In case of any questions, please, contact Aliaksandr Dren in Teams 🙂

### CODE REVIEW

File: authorization-service/src/libs/lambda.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import middy from "@middy/core";
+import middyJsonBodyParser from "@middy/http-json-body-parser";
+
+export const middyfy = (handler) => {
+    return middy(handler).use(middyJsonBodyParser());
```
Comment (DrDiman) [inline authorization-service/src/libs/lambda.ts]: no need to use middleware here at all. As I've understood - authorizer is as middleware by itself and middyJsonBodyParser is redundant

File: authorization-service/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "dependencies": {
+        "@middy/core": "^1.5.2",
+        "@middy/http-json-body-parser": "^1.5.2",
+        "source-map-support": "^0.5.19"
+    },
+    "devDependencies": {
+        "@serverless/typescript": "^2.23.0",
+        "@types/aws-lambda": "^8.10.71",
+        "@types/node": "^14.14.25",
+        "json-schema-to-ts": "^1.5.0",
```
Comment (DrDiman) [inline authorization-service/package.json]: revisit some dependencies. json-schema-to-ts - not used in this service

**Review by DrDiman — COMMENTED** [review]
looks ok. 6 points

File: authorization-service/package.json (hunk 2 above)
Comment (noway36) [reply authorization-service/package.json]: Ye, there were some redundant dependencies, that came up with the `aws-nodejs-typescript` template. The same for `middyfy`. Removed, thanks 🙂

**Pull request merged**